web/src/people.ts

```diff
@@ -1951,6 +1972,111 @@ export function populate_valid_user_ids(params: StateData["user_groups"]): void
     }
 }
 
+function get_combined_promise_for_user_ids(user_ids: Set<number>): {
+    promise_for_user_ids_in_transit: Promise<unknown>;
+    user_ids_pending_fetch: Set<number>;
+} {
+    const promises: Promise<void>[] = [];
+    let user_ids_pending_fetch = new Set<number>(user_ids);
+    for (const [user_ids_set, promise_data] of fetch_users_storage.promise_for_in_transit) {
+        if (user_ids_set.intersection(user_ids).size > 0) {
```

Comment:
`user_ids` here should be `user_ids_pending_fetch`, to avoid making the comparisons more expensive than they need to be, right?

Reply:
Yeah, great point! Changed.